go/oasis-node/cmd/debug/txsource/workload/commission.go

```diff
@@ -35,6 +35,7 @@ const (
 type commission struct {
 	BaseWorkload
 
+	beacon        beacon.Backend
```

Comment:
Could get this from `c.Consensus().Beacon()` now.

Reply:
`./commission.go:135:36: c.BaseWorkload.Consensus().Beacon undefined (type "github.com/oasisprotocol/oasis-core/go/consensus/api".ClientBackend has no field or method Beacon)`